kolibri/core/analytics/utils.py

```diff
@@ -73,9 +73,6 @@ def normalize_version_to_semver(version):
 
     initial, dev = re.match(r"(.*?)(\.dev.*)?$", version).groups()
```

Comment:
if `initial` is not used anymore, I would prefer to make it evident using the format
`_ , dev = re.match(r"(.*?)(\.dev.*)?$", version).groups()`

Reply:
Fair!

Reply:
Updated!